packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -285,18 +286,17 @@ class ImageCache {
     }
   }
 
-  void _trackLiveImage(Object key, _CachedImage image) {
+  void _trackLiveImage(Object key, _LiveImage image, { bool putOk = true }) {
```

Comment:
should putOk be named debugPutOk to indicate that this is only relevant in debug mode?

Reply:
Done